shared/common-adapters/radio-button.native.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React from 'react'
+import ClickableBox from './clickable-box'
+import Text from './text'
+import type {Props} from './radio-button'
+import {globalStyles, globalColors} from '../styles'
+
+export const RADIOBUTTON_SIZE = 22
+export const RADIOBUTTON_MARGIN = 8
+
+const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => {
+  const boxStyle = {
```

Comment:
@chrisnojima Thanks!  Both done, switched over to glamorous on native too.  How's it look now?